crates/issue/src/lib.rs

```diff
@@ -228,6 +232,12 @@ impl<T: Config> Module<T> {
         );
         ext::collateral::lock_collateral::<T>(&requester, griefing_collateral)?;
 
+        // only continue if the payment is above the dust value
+        ensure!(
+            amount_requested >= Self::issue_btc_dust_value(),
```

Comment:
Shouldn't we be comparing `amount_user`? That's the amount of bitcoin to be sent. We do it like that in redeem as well

Reply:
I think `amount_requested` is the actual value transferred in Bitcoin, `amount_user` is just with the fees deducted. Or do you mean that we want this above the dust value?